app/forms/candidate_interface/pick_site_form.rb

```diff
@@ -20,13 +20,17 @@ def save
 
       application_form.application_choices.create!(
         course_option: course_option,
+        current_option: course_option,
       )
     end
 
     def update(application_choice)
       return unless valid?
 
-      application_choice.update!(course_option: course_option)
+      application_choice.update!(
+        course_option: course_option,
+        current_option: course_option,
```

Comment:
I could see a future developer wondering about redundancy here. Maybe give this procedure a name, like `ApplicationChoice#set_initial_course_choice!(course_option)`?